doc/api/fs.md

```diff
@@ -876,6 +876,11 @@ try {
 
 <!-- YAML
 added: v16.7.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/41819
+    description: Accepts an additional `verbatimSymlinks` parameter to specify
```

Comment:
I think `parameter` could mean it's a "function parameter", maybe we should say `option` instead?
```suggestion
    description: Accepts an additional `verbatimSymlinks` option to specify
```

Reply:
Changed!

Reply:
It seems that this hasn't been changed.

Reply:
You're right, sorry. I skipped this particular one. It has been solved now.